scripts/build/testing.mk

```diff
@@ -118,7 +118,7 @@ start-geth-bepolia:
 	--rm -v $(PWD)/${TESTAPP_FILES_DIR}:/${TESTAPP_FILES_DIR} \
 	--rm -v $(PWD)/${BEPOLIA_NETWORK_FILES_DIR}:/${BEPOLIA_NETWORK_FILES_DIR} \
 	-v $(PWD)/.tmp:/.tmp \
-	ethereum/client-go init \
```

Comment:
we reth fork with the original image for bepolia and mainnet. Why this difference?
I see you have synced, so we should be fine. Still noting we should probably do the same thing across the two forks?